.github/workflows/doc.yml

```diff
@@ -0,0 +1,63 @@
+name: doc
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
+  cancel-in-progress: true
+on:
+  pull_request:
+  push:
+    branches: [dev]
```

Comment:
on release, there should probably also be a build of the doc

- dev should always replace the old dev, but release are only ever built once (of the stable branch then).

Reply:
currently (in our other repositories) we build on PRs and pushes to our development branch. That way the documentation always reflects the current dev version. Since doc strings are for developers mainly this makes most sense. In order to store (and not replace) webpages for releases, we would need to setup a more complex procedure, where we create a copy of the webpage from dev into a stable directory. Currently this has to be done manually.